tests/runner.py

```diff
@@ -1651,10 +1651,7 @@ def build_library(name,
         stderr = open(os.path.join(project_dir, 'configure_err'), 'w')
       else:
         stderr = None
-      try:
-        Building.configure(configure + configure_args, env=env, stdout=stdout, stderr=stderr)
-      except subprocess.CalledProcessError:
-        pass # Ignore exit code != 0
+      Building.configure(configure + configure_args, env=env, stdout=stdout, stderr=stderr)
```

Comment:
I think this might change behaviour. Still, "errors should never pass silently" for no reason.